modules/serverless-installing-knative-serving.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// serverless/installing-openshift-serverless.adoc
+
+[id="installing-knative-serving_{context}"]
+= Installing Knative Serving
+
+You must create a `KnativeServing` object to install Knative Serving using the OpenShift Serverless Operator.
+
+[source,yaml]
+.Sample `serving.yaml`
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+ name: knative-serving
+---
+apiVersion: serving.knative.dev/v1alpha1
+kind: KnativeServing
+metadata:
+ name: knative-serving
+ namespace: knative-serving
+----
+
+
+.Prerequisite
+* An account with cluster administrator access.
+* Installed OpenShift Serverless Operator.
+
+
+.Procedure
+. Copy the sample YAML file into `serving.yaml` and apply it using:
++
+----
+$ oc apply -f serving.yaml
+----
++
+. Verify the installation is complete by using the command:
++
+----
+$ oc get knativeserving/knative-serving -n knative-serving --template='{{range .status.conditions}}{{printf "%s=%s\n" .type .status}}{{end}'
```

Comment:
This command is missing one curly bracket at the end. Should be ```oc get knativeserving/knative-serving -n knative-serving --template='{{range .status.conditions}}{{printf "%s=%s\n" .type .status}}{{end}}'```

Reply:
Got it !! Thanks Martin